test/dashboard-e2e/api/api-helpers.js

```diff
@@ -1,67 +1,75 @@
 import superagent from 'superagent';
 
 export class ApiHelpers {
-    constructor(apiUrl) {
-        this.API_URL = apiUrl;
+    constructor(apiUrl, cloudContext, bearerToken) {
+        this.apiUrl = apiUrl;
+        this.cloudContext = cloudContext;
+        this.bearerToken = bearerToken;
     }
 
     async getTests() {
-        const request = `${this.API_URL}/tests`;
+        const request = `${this.apiUrl}/tests`;
 
         try {
-            const response = await superagent.get(request);
+            const response = await superagent.get(request)
+                .set('Content-Type', 'application/json')
+                .set('Authorization', this.cloudContext ? `Bearer ${this.bearerToken}` : '');
```

Comment:
Is the `cloudContext` needed? It could depend on `bearerToken`. Same for other usages.

Reply:
It can depend on bearerToken, but I think using cloudContext specifically make things more readable - especially it's also used in other places

Reply:
Aren't all the cases when the `cloudContext` is used, the cases were the `bearerToken` is needed?